tests/unittests/cli/test_cli.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+
+import pytest
+from click.testing import CliRunner
+
+from scenario_player import main
+from scenario_player.exceptions import ScenarioError
+from scenario_player.exceptions.cli import WrongPassword
+
+
+@pytest.fixture(scope="module")
+def runner():
+    return CliRunner()
+
+
+KEYSTORE_PATH = str(Path(__file__).resolve().parents[0].joinpath("keystore"))
+CLI_ARGS = [
```

Comment:
```suggestion
CLI_ARGS = f"--chain http://geth.goerli.ethnodes.brainbot.com:8545 run --keystore-file {KEYSTORE_PATH.joinpath('UTC--1')} --no-ui  {{pw_option}} {Path(__file__).resolve().parent.joinpath('scenario', 'join-network-scenario-J1.yaml')}"
```

And in tests use `str.format()` instead of list concatenation:

```python
...
result = runner.invoke(main.main, CLI_ARGS.format(pw_option="--password 123").split(" ")
...
```